symbols/arm9.yml

```diff
@@ -1279,30 +1279,381 @@ arm9:
+    - name: GetNbFriendRescues
+      address:
+        NA: 0x204FCC8
+        EU: 0x2050000
+        JP: 0x2050014
+      description: |-
+        Gets the number of successful friend rescues.
+        
+        return: the number of successful friend rescues
+    - name: IncrementNbEvolutions
+      address:
+        NA: 0x204FCDC
+        EU: 0x2050014
+        JP: 0x2050028
+      description: |-
+        Increments by 1 the number of evolutions.
+        
+        No params.
+    - name: GetEvolutions
+      address:
+        NA: 0x204FD24
+        EU: 0x205005C
+        JP: 0x2050070
+      description: |-
+        Gets the number of evolutions.
+        
+        return: the number of evolutions
+    - name: IncrementNbSteals
+      address:
+        NA: 0x204FD38
+        EU: 0x2050070
+        JP: 0x2050084
+      description: |-
+        Leftover from Time & Darkness. Does not do anything.
+        
+        Calls to this matches the ones for incrementing the number of successful steals in Time & Darkness.
+        
+        No params.
+    - name: IncrementNbEggsHatched
+      address:
+        NA: 0x204FD3C
+        EU: 0x2050074
+        JP: 0x2050088
+      description: |-
+        Increments by 1 the number of eggs hatched.
+        
+        No params.
+    - name: GetNbEggsHatched
+      address:
+        NA: 0x204FD78
+        EU: 0x20500B0
+        JP: 0x20500C4
+      description: |-
+        Gets the number of eggs hatched.
+        
+        return: the number of eggs hatched
+    - name: GetNbPokemonJoined
+      address:
+        NA: 0x204FD8C
+        EU: 0x20500C4
+        JP: 0x20500D8
+      description: |-
+        Gets the number of different pokémon that joined.
+        
+        return: the number of different pokémon that joined
+    - name: GetNbMovesLearned
+      address:
+        NA: 0x204FDA0
+        EU: 0x20500D8
+        JP: 0x20500EC
+      description: |-
+        Gets the number of different moves learned.
+        
+        return: the number of different moves learned
+    - name: SetVictoriesOnOneFloor
+      address:
+        NA: 0x204FDB4
+        EU: 0x20500EC
+        JP: 0x2050100
+      description: |-
+        Sets the record of victories on one floor.
+        
+        r0: the new record of victories
+    - name: GetVictoriesOnOneFloor
+      address:
+        NA: 0x204FDE8
+        EU: 0x2050120
+        JP: 0x2050134
+      description: |-
+        Gets the record of victories on one floor.
+        
+        return: the record of victories
+    - name: SetPokemonJoined
+      address:
+        NA: 0x204FDFC
+        EU: 0x2050134
+        JP: 0x2050148
+      description: |-
+        Marks one pokémon as joined.
+        
+        r0: monster ID
+    - name: SetPokemonBattled
+      address:
+        NA: 0x204FE58
+        EU: 0x2050190
+        JP: 0x20501A4
+      description: |-
+        Marks one pokémon as battled.
+        
+        r0: monster ID
+    - name: GetNbPkmnBattled
```

Comment:
For consistency with `GetNbPokemonJoined`, let's spell out `GetNbPokemonBattled` fully here too (and actually you mention "GetNbPokemonBattled" in a later description anyway).